Taybullpay/tests/registration.robot

```diff
@@ -35,14 +41,12 @@ Verify That User Cannot Register Using An Incorrect Email Address
     [Teardown]    Close Browser
 
 
-#Verify That User Can Login Using A Single Email
-# Precondition: 1. Open Browser
-#               2. Navigate to the TaybullPay page
-# Steps:
-#               1. Go To Registration Page
-#               2. Fill in a single name
-#               3. Fill in  a valid Email Address
-#               4. Provide Strong Password
-#               5  Click on Sign Up
-# Expected Result
-#   Opens a new page that contains "Verify your phone number with a code"
```

Comment:
I think these information aren't necessary here. Is it possible the project moves along without too many stories?